flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/DefaultExecutionGraph.java

```diff
@@ -364,6 +373,8 @@ public DefaultExecutionGraph(
         this.resultPartitionsById = new HashMap<>();
 
         this.isDynamic = isDynamic;
+
+        LOG.info("Created execution graph {}.", executionGraphId);
```

Comment:
This should also log the job id. Let's have 1 line that couples JobID<->ExecutionGraphID, and remove the need to also read previous log messages.

Reply:
Agreed.

Reply:
done.